.gometalinter.json

```diff
@@ -0,0 +1,25 @@
+{
+	"DisableAll": true,
+	"Enable": [
+		"deadcode",
+		"dupl",
+		"errcheck",
+		"gas",
+		"goconst",
+		"gocyclo",
+		"goimports",
+		"golint",
+		"gotype",
+		"gotypex",
+		"ineffassign",
+		"maligned",
+		"megacheck",
+		"misspell",
+		"nakedret",
+		"safesql",
+		"structcheck",
+		"unconvert",
+		"varcheck",
+		"vet"
+  ]
```

Comment:
Maybe this `]` can be aligned?
I think spaces should be replaced with tab there.
You can turn on whitespace display in your editor. Makes this issues obvious.

Reply:
Done.